src/styles/index.scss

```diff
@@ -1,3 +1,5 @@
-body {
-  margin: 0;
-}
+@import 'blocks/veriables';
```

Comment:
There is a typo in the import statement. It should be `@import 'blocks/variables';` instead of `@import 'blocks/veriables';`.